src/test/resources/com/jpinpoint/perf/lang/java/ruleset/concurrent/xml/AvoidParallelFlux.xml

```diff
@@ -52,4 +52,33 @@ class Account {
        ]]></code>
     </test-code>
 
+
+    <test-code>
+        <description>Violation: Fix Request: AvoidParallelFlux missed case #516</description>
+        <expected-problems>1</expected-problems>
+        <expected-linenumbers>9</expected-linenumbers>
+        <code><![CDATA[
+import java.util.*;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Scheduler;
+
+class FooBad {
+    private final Scheduler scheduler;
+    private final SchedulerProperties schedulerProperties;
+    public Flux<Account> getResponseAccounts(List<AccountDetails> accountKeys, List<FieldName> requestedFields) {
+        return Flux.fromIterable(accountKeys)
+                .parallel() // bad - default parallelism - missed case
```

Comment:
I see there is also: [parallel](https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html#parallel-int-int-)(int parallelism, int prefetch) with three arguments, guess that will be matched as well now, which is good?

Reply:
ah, good. Yes, I believe that is good. Never seen it used btw.